.github/workflows/build.yml

```diff
@@ -7,22 +7,38 @@ jobs:
         java-version: [1.8, 11, 14]
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - uses: gradle/wrapper-validation-action@v1
       - uses: actions/setup-java@v1
         with:
           java-version: ${{ matrix.java-version }}
       - uses: eskatos/gradle-command-action@v1.3.2
         with:
           gradle-version: wrapper
-          arguments: check build publishToMavenLocal --stacktrace --parallel
+          arguments: remapAllJars compileTestmodJava --stacktrace --parallel
```

Comment:
No, im not a fan of this. The idea with this workflow is to check that the full process completes, including publishing to a maven.

We also decided `--parallel` is pointless on GHA

Reply:
I was mistaken and parallel does have a small benefit